docs/ml/vision/_index.md

```diff
@@ -18,23 +22,83 @@ While the camera component lets you access what your machine's camera sees, the
 
-{{< cards >}}
-{{% card link="/ml/vision/detection/" %}}
-{{% card link="/ml/vision/classification/" %}}
-{{% card link="/ml/vision/segmentation" %}}
-{{< /cards >}}
+- [Detections](#detections)
+- [Classificaitons](#classifications)
+- [Segmentations](#segmentations)
+
+## Detections
+
+<div class="td-max-width-on-larger-screens">
+  <div class="alignright" >
+    {{< imgproc alt="A white dog with a bounding box around it labeled 'Dog: 0.71'" src="/ml/vision/dog-detector.png" resize="300x" declaredimensions=true >}}
+  </div>
+</div>
+
+_2D Object Detection_ is the process of taking a 2D image from a camera and identifying and drawing a box around the distinct "objects" of interest in the scene.
+Any camera that can return 2D images can use 2D object detection.
+
+The service provides different types of detectors, both based on heuristics and machine learning, so that you can create, register, and use detectors for any object you may need to identify.
+
+The returned detections consist of the bounding box around the identified object, as well as its label and confidence score:
+
+- `x_min`, `y_min`, `x_max`, `y_max` (int): specify the bounding box around the object.
+- `class_name` (string): specifies the label of the found object.
+- `confidence` (float): specifies the confidence of the assigned label.
+  Between `0.0` and `1.0`, inclusive.
+
+## Classifications
+
+_2D Image Classification_ is the process of taking a 2D image from a camera and deciding which class label, out of many, best describes the given image.
+Any camera that can return 2D images can use 2D image classification.
+
+The class labels used for classification vary and depend on the machine learning model and how it was trained.
+
+The returned classifications consist of the image's class label and confidence score.
+
+- `class_name` (string): specifies the label of the found object.
+- `confidence` (float): specifies the confidence of the assigned label.
+  Between `0.0` and `1.0`, inclusive.
+
+## Segmentations
+
+_3D Object Segmentation_ is the process of separating and returning a list of the identified "objects" from a 3D scene.
+The "objects" are usually a list of point clouds with associated metadata, like the label, the 3D bounding box, and center coordinates of the object.
+
+3D object segmentation is useful for obstacle detection.
+See our guide [Navigate with a Rover Base](/tutorials/services/navigate-with-rover-base/#next-steps-automate-obstacle-detection) for an example of automating obstacle avoidance with 3D object segmentation for obstacle detection.
+
+Any camera that can return 3D pointclouds can use 3D object segmentation.
+
+## Supported Models
+
+To use your motor with Viam, check whether one of the following [built-in models](#built-in-models) or [modular resources](#modular-resources) supports your motor.
+
+{{< readfile "/static/include/create-your-own-mr.md" >}}
+
+### Built-in models
+
+For configuration information, click on the model name:
```

Comment:
(opt) Adding a column like `| Type | `  ` | _detector_ | ` would be helpful IMO. For most it is obvious but I am confused on those that are seemingly both segmenters/detectors (are those separate things?)

Reply:
You mean the one that uses a detector to turn it into segmenter output. It is a segmenter. I can't think how to make the description better

Reply:
Got it, makes sense. And yeah, maybe